docs/encyclopedia/nexus-operations.mdx

```diff
@@ -0,0 +1,184 @@
+
+### System interactions
+
+Nexus Operations are often registered in the same Worker as the underlying Temporal primitives they abstract.
+
+![NexusYourCloud](/img/nexus/nexus-connect-queue-based-workers.png)
+
+For example, when you execute a Nexus Operation in a caller Workflow:
+1. **RespondWorkflowTaskCompleted([ScheduleNexusOperation command](/references/commands#schedulenexusoperation))** \-
+is used by the caller Worker to schedule a Nexus Operation, which atomically hands off execution to the caller's Nexus Machinery.
+2. **PollNexusTaskQueue** \-
+is used by the handler Worker to recieve a [Nexus Task](/workers#nexus-task) to process, for example to start a Nexus Operation.
+3. **RespondNexusTaskCompleted** or **RespondNexusTaskFailed** \-
+is used by the handler Worker to return the Nexus Task result.
+When asked to start a Nexus Operation, the Nexus handler decides if the Operation will be synchronous or asynchronous.
+   1. This is typically a static decision based on the [Temporal SDK builder function used](#sdk-support).
+   2. [Asynchronous Nexus Operations](#asynchronous-operation-lifecycle), created with the New-Workflow-Run-Operation SDK helper, will return a Nexus Operation ID, that can be used to perform additional actions like canceling an Operation.
+   3. [Synchronous Nexus Operations](#synchronous-operation-lifecycle), created with the New-Sync-Operation SDK helper, will return the Operation result directly.
+The caller's Nexus Machinery recieves the result and records a [NexusOperation event](/references/events#nexusoperationstarted) in the caller's Workflow History.
+4. **PollWorkflowTaskQueue** \-
+is used by the caller Worker to recieve a Workflow Task with the [Nexus Operation event](/references/events#nexusoperationstarted) which may be Started, Completed, Failed, Canceled, or TimedOut.
+
+### Asynchronous Operation lifecycle {#asynchronous-operation-lifecycle}
+
+An asynchronous Nexus Operation may take up to 60 days to complete in Temporal Cloud, to allow for callback URL token validation with asymmetric key rotation at those intervals.
+
+The lifecycle of an asynchronous Nexus Operation:
+
+1. Caller’s Workflow executes a Nexus Operation using the Temporal SDK.
+   1. `NexusOperationScheduled` is added to caller’s Event History.
+   2. Nexus Operation added to Nexus Machinery in the caller’s Temporal cluster.
+   3. Caller Workflow doesn’t automatically block on a Nexus Operation.
+   4. Caller’s Workflow can interact with the Nexus Operation execution:
+      1. Wait for the Nexus Operation to start.
+      2. Wait for the Operation result.
+      3. Cancel the Nexus Operation.
+2. The caller’s Temporal Service attempts to start the Nexus Operation on the target cluster through the Nexus Endpoint.
+   1. Nexus Machinery in the target cluster sync matches the Nexus request to a Worker polling the endpoint's target Namespace and Task Queue.
```

Comment:
Maybe use handler instead of target for consistency?

Reply:
will use handler for the other occurrences of target in the section, but the `Endpoint` has a target, so think we should keep using that here.